schemas/storage.sql

```diff
@@ -374,3 +374,49 @@ WHERE StarRecords.StorageSystemID = StorageSystems.id
   AND StarRecords.GroupID = Groups.id
   AND StarRecords.SubGroupID = SubGroups.id
   AND StarRecords.RoleID = Roles.id;
+
+-- -----------------------------------------------------------------------------
+-- DaySummaries
+DROP TABLE IF EXISTS DaySummaries;
+CREATE TABLE DaySummaries (
+  Site                        VARCHAR(255),
+  StorageSystem               VARCHAR(255),
+  StorageShare                VARCHAR(255),
+  StorageMedia                VARCHAR(255),
+  DirectoryPath               VARCHAR(255),
+  `Group`                     VARCHAR(255),
```

Comment:
Why the backticks here? Codacy doesn't seem to like it

Reply:
We might need to give more priority to SQL here than Codacy. @rowan04, As we wanted column name to be Group. We are using as `Group` because since it is a reserve keyword. We cannot directly use keywords because it might throw syntax errors or confuses SQL.